library/src/com/google/maps/android/geojson/GeoJsonGeometryCollection.java

```diff
@@ -0,0 +1,53 @@
+package com.google.maps.android.geojson;
+
+import java.util.ArrayList;
+
+/**
+ * A GeoJsonGeometryCollection geometry contains a number of GeoJsonGeometry objects.
+ */
+public class GeoJsonGeometryCollection implements GeoJsonGeometry {
+
+    private final static String GEOMETRY_TYPE = "GeometryCollection";
+
+    private final ArrayList<GeoJsonGeometry> mGeometries;
+
+    /**
+     * Creates a new GeoJsonGeometryCollection object
+     *
+     * @param geometries array of GeoJsonGeometry objects to add to the GeoJsonGeometryCollection
```

Comment:
nit: 'array of ...' does not match the method signature, which requires an ArrayList. You also say "to add to the ...", which implies that it's additive, which it is not.